themes/bootstrap3/templates/myresearch/checkedout.phtml

```diff
@@ -29,13 +29,14 @@
       <? endif; ?>
 
       <? if ($paginator): ?>
-        <?=$this->transEsc("Showing")?>
-        <? $start = $paginator->getAbsoluteItemNumber(1);
-           $end = $paginator->getAbsoluteItemNumber($paginator->getItemCountPerPage());
-           $total = $paginator->getTotalItemCount();
+        <?
+          $transParams = [
+            '%%start%%' => $this->localizedNumber($paginator->getAbsoluteItemNumber(1)),
+            '%%end%%'   => $this->localizedNumber($paginator->getAbsoluteItemNumber($paginator->getItemCountPerPage())),
```

Comment:
The old code constrained this, so end was never greater than total. That constraint seems to be lost here, which allows results like "showing 5-6 of 5". Can you restore the old constraint, and make sure that this same bug has not been duplicated in other refactored templates?